### PR TITLE
move the Unused.hpp file from opm-material to here

### PR DESCRIPTION
**Pull request by andlaus**
this PR requires quite a bit of mob-up PRs in other modules. make sure to also merge these.

### CODE REVIEW

**Comment by andlaus** [comment]
ok, the three mob-ups are OPM/opm-material#179, OPM/ewoms#92 and OPM/opm-upscaling#211.

File: opm/common/Unused.hpp [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+  OPM is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+
+  You should have received a copy of the GNU General Public License
+  along with OPM.  If not, see <http://www.gnu.org/licenses/>.
+
+  Consult the COPYING file in the top-level source directory of this
```
Comment (atgeirr) [inline opm/common/Unused.hpp]: There is no such file, there is a LICENSE file though.

Comment (andlaus) [inline opm/common/Unused.hpp]: yeah, that was caused by the fact that the file is a verbatim copy of what was in opm-material. fixed. thanks.

**Comment by andlaus** [comment]
@atgeirr: can this go in?

**Comment by atgeirr** [comment]
Yes I think it's ok, saw now that you had updated it long ago... Merging.

**Comment by atgeirr** [comment]
jenkins build this opm-material=179 ewoms=92 opm-upscaling=211 with downstreams please

**Comment by atgeirr** [comment]
The Jenkins failure is an ewoms thing, can you take a look at that @andlaus?

**Comment by andlaus** [comment]
looks like something went wrong on the jenkins side. the first error it spits out is

```
/var/lib/jenkins/workspace/opm-common-PR-builder/deps/ewoms/ebos/eclproblem.hh:1073:53: error: 'phaseIsActive' is not a member of 'Ewoms::EclProblem<Ewoms::Properties::TTag::EclProblem>::FluidSystem {aka Opm::FluidSystems::BlackOil<double>}'
```

which is a method that was recently introduced by the 2p stuff. I've checked: it is there (https://github.com/OPM/opm-material/blob/master/opm/material/fluidsystems/BlackOilFluidSystem.hpp#L403 ) and everything in ebos compiles locally on my machine. maybe retriggering Jenkins helps.

**Comment by andlaus** [comment]
jenkins build this opm-material=179 ewoms=92 opm-upscaling=211 with downstreams please

**Comment by akva2** [comment]
the only difference between the failed run and the successful run that i can find, is that opm-material was rewritten. did you forget to push something or rebase or something? the only alternative i can think of is that github somehow takes time to process pull refs, though that i have never seen that before.

**Comment by andlaus** [comment]
> did you forget to push something or rebase or something?

I didn't push anything after @atgeirr triggered jenkins the first time. maybe its a github bug. (this wouldn't be the first one we encounter.)

anyway, seems like everything is okay now...

**Comment by atgeirr** [comment]
> anyway, seems like everything is okay now...

OK, I'll merge it all. Let's be alert for any trouble just in case.

**Pull request merged**